client/src/Components/Layouts/LangingPge/LangingPge.test.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
```

Comment:
Please rename file and folder to correct spelling:

LandingPage

Reply:
I shouldn't code in the dark anymore :smile: